AdditionalText.md

```diff
@@ -16,3 +16,21 @@ We the Core Working Group establish on 2020-12-28 that the Language being develo
 * All orders listed above are to be considered to have the authority of this Governance and any amendment thereof. The Core Working Group shall not dipose of any order listed above, except by amendment.
 * Any text in any existing paper which refers to the Language by the name Laser shall be deemed to use the name Cannon instead. The Responsible Working Group chair shall, in a timely manner, submit a revised version with changes to this effect for approval, unless the Project Manager or the Core Working Group, by *simple-majority* deem it unnecessary for any specific paper. 
 
+## Working Group Suspension
+
+
+1. The Working Group System described herein shall be suspended upon adoption of this amendment, and authority for the Design and Specification of the Cannon Language and Library, and development of the Cannon Compiler shall be transfered to the Community as a whole.
+2. Notwithstanding (1), the Core Working Group, including it's membership at the date of adoption of this amendment, any members admitted by the Project Manager since it's adoption, and any positions held by the membership therein, shall continue to operate as described herein, and in particular shall continue to hold monthly meetings.
+3. (1) shall become void after the meeting during the 6th month following the adoption of this amendment, unless continuation is approved by a simple-majority. If continuation is approved, then the same shall apply, unless a simple-majority sees fit to impose a shorter time limit, but two-thirds majority shall be required to impose a longer limit.
+4. The Core Working Group may, by approval of a simple-majority, cause (1) to become void at any time.
+5. At the expiry of this amendment as prescribed by (3), or on application of (4), all changes made to the specification of the Cannon Language following the date of adoption, shall be repealed and presented by the Project Manager in a timely manner to the Core Working Group for approval as a technical specification, and shall be restored to the specification upon approval.
+6. For greater certainty, until (1) becomes void, the Core Working Group shall nonetheless retain the following authorities, in addition to those explicitly granted by this amendment:
```

Comment:
The list should be exhaustive. If there are any I've missed, they should be added.